recipes/bob.extension/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+PYTHONDONTWRITEBYTECODE=1 $PYTHON setup.py install
```

Comment:
Sorry, why does it need `--single-version-externally-managed --record record.txt`?